src/InstagramClone.Api/appsettings.json

```diff
@@ -6,7 +6,12 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnectionString": "Server=.;Database=InstagramClone2;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnectionString": "Server=.;Database=InstagramClone3;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
you should drop all these old database versions and re-run your migration so that this stay as `InstagramClone`

Reply:
Done.